### PR TITLE
Parameterized SMTP details and admin password

### PR DESCRIPTION
**Pull request by gs11**
Fixes https://github.com/gravitee-io/issues/issues/1600

### CODE REVIEW

File: gravitee/values.yaml [hunk 1]
```diff
@@ -25,6 +25,19 @@ oauth:
 inMemoryAuth:
   enabled: true
 
+# Default password "admin", use bcrypt to generate a new one
+adminPasswordBcrypt: aaa
```
Comment (brasseld) [inline gravitee/values.yaml]: I think it would be useful to keep the default value from `gravitee.yml` here.

Comment (brasseld) [inline gravitee/values.yaml]:
```suggestion
adminPasswordBcrypt: $2a$10$Ihk05VSds5rUSgMdsMVi9OKMIx2yUvMz7y9VP3rJmQeizZLrhLMyq
```

File: gravitee/values.yaml [hunk 2]
```diff
@@ -25,6 +25,19 @@ oauth:
...
+
+smtp:
+  host: smtp.example.com
```
Comment (brasseld) [inline gravitee/values.yaml]: Why not also providing the item for `enabled` property ?

Comment (brasseld) [inline gravitee/values.yaml]:
```suggestion
  enabled: true
  host: smtp.example.com
```

File: gravitee/values.yaml (hunk 1 above)
Comment (gs11) [reply gravitee/values.yaml]: Sorry, leftover from testing. Fixed.

File: gravitee/values.yaml (hunk 2 above)
Comment (gs11) [reply gravitee/values.yaml]: Fixed

**Comment by gs11** [comment]
All suggested changes implemented, thanks for reviewing!

**Comment by brasseld** [comment]
Please squash your commits into a single one.
Once done, I will merge the PR.

**Comment by gs11** [comment]
Done @brasseld

**Comment by brasseld** [comment]
Thanks a lot for your contribution!

**Pull request merged**